### DISCUSSION TITLE
Provide visual tips for Pinned items key bindings

### DISCUSSION THREAD

**Discussion by hereafter** [hereafter/power-start-menu#17] [Ideas]
It is hard to locate the position of pinned apps

It is better if we can have visual indicator one the WIN key is held down for a while

**Comment by ShyRaziel** [comment]
Hi, I bought Power Start Menu but it no longer appears in the Microsoft Store, is there any way you could share the app I paid for? :( Regards and thanks

**Reply by hereafter** [reply]
let me check, microsoft may have pulled it due to some reason

**Reply by hereafter** [reply]
My account has been removed somehow.

**Reply by ShyRaziel** [reply]
> Mi cuenta ha sido eliminada de alguna manera.

So, I'll no longer be able to access the app? Is there any way I could share the app privately with you if I show you a screenshot of my purchase so I can install it on my Windows devices? Thanks for responding.
